src/applications/personalization/rated-disabilities/components/RatedDisabilityListItem.jsx

```diff
@@ -17,7 +17,7 @@ const RatedDisabilityListItem = ({ ratedDisability }) => {
             {name}
           </p>
         </div>
-        {ratingPercentage ? (
+        {ratingPercentage !== null ? (
```

Comment:
Could `ratingPercentage` be something like `undefined`? If we want to make sure 0 is accepted, would `typeof ratingPercentage === 'number'` work?

Reply:
That's a good question. As far as I know, the most it could be is `null` in terms of what's passed to us from EVSS. But, I think `typeof` is better here. I'll update it.